models/nc-news.models.js

```diff
@@ -16,9 +16,12 @@ exports.fetchArticles = (articleId) => {
   return db
     .query(
       `
-    SELECT articles.article_id, title, users.name AS author, body, topic, created_at, votes FROM articles
+    SELECT articles.article_id, articles.title, users.name AS author, articles.body, articles.topic, articles.created_at, articles.votes, COUNT(comments.article_id)::int AS comment_count
+    FROM articles
     JOIN users ON articles.author = users.username
-    WHERE article_id = $1;
+    JOIN comments ON comments.article_id = articles.article_id
```

Comment:
i'd think about this join. At the moment, would this account of articles which don't have comments?